core/sequencer/sequencer.go

```diff
@@ -373,3 +445,21 @@ func queueLogLeaf(ctx context.Context, tlog trillian.TrillianLogClient, logID in
 	}
 	return nil
 }
+
+// RegisterMutationsChannel registers a mutations channel created by the
+// sequencer service handler.
+func (s *Sequencer) RegisterMutationsChannel(ch chan *tpb.GetMutationsResponse) {
```

Comment:
What we're doing here is classic pub/sub.  Might be nice to
a) create a type or package to contain the logic
b) give them standard method names: Register, Unregister, Publish. [example](https://godoc.org/github.com/dustin/go-broadcast)